BiweeklyMeetings/2022-08-02.md

```diff
@@ -0,0 +1,323 @@
+
+**Michael**-: To become the best version of itself.
+
+**Matthias** -: Yes. Let's maybe quote that under the CIP title.
+
+**Matthias** -: Okay. So let's keep the discussions going also and have CIP 50 back for review next week. We haven't got any updates from IOG, think they have been busy with Vasil with some of the recent problem and [inaudible 00:35:06] found. So I haven't heard of Kevin on that. I think Christoff mentioned that he... I mean Kevin said something recently online on the 360 or something like that, but it was nothing really substantial, just that they were looking at it. They were aware that it existed, wanted to well provide also [inaudible 00:35:30]. We'll still a bit waiting on that sentence. So let's see next meeting.
+
+**Anthony**-: Amazing. Great.
+
+#### PR296
+[PR296 CIP-0062? | Governance API for dApp Connectors (dApps<->Wallet Web-bridge](https://github.com/cardano-foundation/CIPs/pull/296)
+
+**Matthias** -: CIP 62 is next in line, right?
+
+**Anthony**-: That's correct.
+
+**Matthias** -: I've been trying to click on the link for 15 seconds and I just realized it was a stream. So can click on the video, obviously. Governance API for DApp, which was proposed by [inaudible 00:36:17] Martins, who is now the tech lead regarding Wallets at IOG correctly. And this is a proposal to extend CIP 30 with capabilities that would make that voting process a bit more streamlined. Sorry, I did a first run of [inaudible 00:36:42] on that one this morning and mainly suggested to simplify a bit some of the methods in particular to make things a bit more [inaudible 00:36:54], meaning not to have the API provide five different ways of doing the same things or in that case two different ways.
+
+**Matthias** -: And I'm also asking for a rational section mainly because we need a rational section for any CIPs in general, because that what exists the discussion for all the bases for discussion and that explains, at a later time, while the decisions were made, the waiting were made. And for that one in particular, because there are a few suggested API or endpoints, you could say, or interfaces, that can be done currently through existing libraries and I don't see why adding them to the bridge, the wallet interface, so to speak makes sense. Or at least there is no justification for it. I can get that it's maybe a bit more simple and that removes some of the burden from the apps implementing that, but those apps are mainly Wallets anyway. So that was a bit some form of the gap in the justification here that I'm liking. I don't know, Sebastien, if you had a look at that yet.
```

Comment:
```suggestion
**Matthias** -: And I'm also asking for a rationale section mainly because we need a rationale section for any CIPs in general, because that what exists the discussion for all the bases for discussion and that explains, at a later time, while the decisions were made, the waiting were made. And for that one in particular, because there are a few suggested API or endpoints, you could say, or interfaces, that can be done currently through existing libraries and I don't see why adding them to the bridge, the wallet interface, so to speak makes sense. Or at least there is no justification for it. I can get that it's maybe a bit more simple and that removes some of the burden from the apps implementing that, but those apps are mainly Wallets anyway. So that was a bit some form of the gap in the justification here that I'm liking. I don't know, Sebastien, if you had a look at that yet.
```
"the waiting were made" = ???